src/vs/workbench/contrib/extensions/browser/extensions.contribution.ts

```diff
@@ -1572,6 +1572,21 @@ class ExtensionsContributions extends Disposable implements IWorkbenchContributi
 			run: async (accessor: ServicesAccessor, id: string) => accessor.get(IClipboardService).writeText(id)
 		});
 
+		this.registerExtensionAction({
+			id: 'workbench.extensions.action.copyMarketplaceLink',
+			title: localize2('workbench.extensions.action.copyMarketplaceLink', 'Copy Marketplace Link'),
+			menu: {
+				id: MenuId.ExtensionContext,
+				group: '1_copy',
+				when: ContextKeyExpr.and(ContextKeyExpr.not('isBuiltinExtension'), ContextKeyExpr.not('extensionDisallowInstall')),
+			},
+			run: async (accessor: ServicesAccessor, id: string) => {
+				const clipboardService = accessor.get(IClipboardService);
+				const extensionLinkPrefix = 'https://marketplace.visualstudio.com/items?itemName=';
```

Comment:
You need to get the marketplace link from the productService and not to hard code it.

Reply:
I have now fixed that issue.